core/modules/saver-handler.js

```diff
@@ -125,15 +125,22 @@ SaverHandler.prototype.initSavers = function(moduleType) {
 	var self = this;
 	$tw.modules.forEachModuleOfType(moduleType,function(title,module) {
 		if(module.canSave(self)) {
-			self.savers.push(module.create(self.wiki));
+			self.savers.push({title:title,module:module.create(self.wiki)});
```

Comment:
This change to the "savers" property isn't backwards compatible, is there another way to approach this?

Reply:
If we can iterate the savers module array to either locate the saver title of info.name it could be avoided.

Reply:
Could you elaborate a little more about what you mean with 'backward compatible' ? It sound to me that this data structure is internal.